kubernetes/structure_service_spec.go

```diff
@@ -138,6 +177,30 @@ func expandIPFamilies(l []interface{}) []v1.IPFamily {
 	return obj
 }
 
+func expandSessionAffinityConfigClientIP(l []interface{}) *v1.ClientIPConfig {
+	var to int32
+	in := l[0].(map[string]interface{})
+	if v, ok := in["timeout_seconds"].(int); ok {
+		to = int32(v)
+	}
+	return &v1.ClientIPConfig{TimeoutSeconds: &to}
```

Comment:
I would put this statement as a return inside the if statement as above, an then return `&v1.ClientIPConfig{}` here as the default case.

Reply:
Thanks, @jrhouston! I like this suggestion, I've made some optimizations here.